src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataResourceDeserializer.cs

```diff
@@ -245,6 +265,20 @@ public virtual object CreateResourceInstance(IEdmStructuredTypeReference structu
             }
         }
 
+        private static void AppendDeletedProperties(dynamic resource, ODataDeletedResource deletedResource, bool isUntyped)
+        {
+            if (isUntyped)
```

Comment:
>if (isUntyped) [](http://example.com/codeflow?start=12&length=14)

why is logic different for untyped than typed?

Reply:
One is string vs another is Uri